Sources/AsyncHTTPClient/AsyncAwait/HTTPClientRequest.swift

```diff
@@ -341,7 +341,13 @@ extension HTTPClientRequest.Body {
         public static let unknown: Self = .init(storage: .unknown)
 
         /// The size of the request body is known and exactly `count` bytes
+        @available(*, deprecated, message: "Use `known(_ count: Int64)` instead")
```

Comment:
I think this message can be more helpful: let's explicitly point out we need the argument to be Int64 instead of Int.